src/ring_theory/simple_module.lean

```diff
@@ -91,6 +93,38 @@ theorem is_semisimple_iff_top_eq_Sup_simples :
   Sup {m : submodule R M | is_simple_module R m} = ⊤ ↔ is_semisimple_module R M :=
 ⟨is_semisimple_of_Sup_simples_eq_top, by { introI, exact is_semisimple_module.Sup_simples_eq_top }⟩
 
+section category
+open category_theory
+open Module
+
+lemma subsingleton_of_epi_zero (N : Module R) [epi (0 : N ⟶ of R M)] : subsingleton M :=
+begin
+  apply subsingleton_of_forall_eq (0 : M),
+  have : function.surjective (0 : N ⟶ of R M) := (Module.epi_iff_surjective _).mp ‹_›,
+  exact this.forall.mpr (λ _, rfl),
+end
+
+instance is_simple_module_of [_inst : is_simple_module R M] : is_simple_module R (of R M) :=
+_inst
+
+/-- A simple module is a simple object in the category of modules. -/
+instance simple_of_is_simple_module [is_simple_module R M] : simple (of R M) :=
+{ mono_is_iso_iff_nonzero := λ N f inj, begin
+    split,
+    { unfreezingI { rintro h rfl },
+      haveI : subsingleton M := subsingleton_of_epi_zero N,
+      haveI : nontrivial M := is_simple_module.nontrivial R M,
+      exact false_of_nontrivial_of_subsingleton M },
+    { intro h,
+      haveI : epi f,
+      { rw epi_iff_range_eq_top,
+        refine (eq_bot_or_eq_top f.range).resolve_left _,
+        exact (mt linear_map.range_eq_bot.mp h)},
+      exact is_iso_of_mono_of_epi _ }
+  end }
+
+end category
```

Comment:
I'd argue that this stuff belongs somewhere in the `algebra.category.Module` folder, rather than making this part of mathlib depend on category theory.

Reply:
For some reason I'm not able/don't know how to create a new file in the PR so I wasn't able to move this to a new file in algebra/category/Module

Reply:
How are you using `git` at the moment?

Reply:
I'm using instructions at https://leanprover-community.github.io/contribute/index.html#making-a-pull-request-pr,  adding new commits by git commit -a + git push origin.

On VSCode it says "untracked file" for my new file if it helps

Reply:
`git add yourfile.lean` before the commit step will do the trick

Reply:
Thanks !